src/structure/widget.ts

```diff
@@ -809,6 +819,7 @@ export class JSmolWidget {
         }
     }
 
+    /// Background *atoms*
```

Comment:
Feel free to rename the functions (since it is a private function) for something more descriptive!

Reply:
This was more of a note to self, because I found the name confusing. I can remove it from this PR and make a separate, small PR to change the name.

Reply:
Sounds good!